setup.cfg

```diff
@@ -45,12 +45,3 @@ exclude =
 
 [bdist_wheel]
 universal = 1
-
-[flake8]
-max-complexity = 10
-max-line-length = 119
-exclude = .venv,djstripe/migrations/,.tox
-
-[pycodestyle]
-max-line-length = 119
-exclude = .venv,djstripe/migrations/,.tox
```

Comment:
Go ahead and keep this in. I'll be updating manage.py tonight to not cause everything to fail if coverage or pep8 fails.